ocs_ci/framework/conf/default_config.yaml

```diff
@@ -92,6 +92,7 @@ REPORTING:
   default_ocs_must_gather_latest_tag: 'latest-4.6'
   gather_on_deploy_failure: true
   gather_on_deploy_success: true
+  collect_logs_on_success_run: False
```

Comment:
Jim I think it's ok to have _run here. But I don't mind to remove _run from here as well.. but if you will remove it from here, we should remove also from CLI arg.  @vavuthu @dahorak @clacroix12 what do you like more? :)

Reply:
Felt like it sounded a little unnecessary once we removed it under the REPORTING from the RUN :)
But we can leave it.

Reply:
I think removing `-run` and `_run` sounds better but this can be done afterwards as a follow up.

As a related question, I'm wondering if we need two separate CLI args for the log collection. Would it be better to have one like `--collect-logs` that takes a string as an argument like `failure|success|both` in order to configure when we want to run must gather? We could keep the separate config variables that would be manipulated by the one CLI argument.

Full examples:

```
--collect-logs failure
--collect-logs success
--collect-logs both
```

WDYT?

Reply:
That sounds like a great solution for this